mne/utils/docs.py

```diff
@@ -1199,9 +1199,6 @@ class deprecated(object):
     # Adapted from http://wiki.python.org/moin/PythonDecoratorLibrary,
     # but with many changes.
 
-    # scikit-learn will not import on all platforms b/c it can be
-    # sklearn or scikits.learn, so a self-contained example is used above
-
```

Comment:
I can't figure out what this note was referring to, so I just removed it.  If someone knows, I can restore it if necessary.